### PR TITLE
[CCP-221] Changes for selecting the rows in datable.

### PR DESCRIPTION
**Pull request by mkarthik90**

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
ok, but we now need the word field to be editable though. If I select two entries, I want to be able to edit the word that is used to store the concept.

**Comment by jdamerow** [comment]
and apparently wordnet ids are not stored in the new concept:
image

**Comment by jdamerow** [comment]
Hm, I guess there are stored but not displayed.

**Comment by mkarthik90** [comment]
@jdamerow Addressed review comments.

**Comment by jdamerow** [comment]
word field is still not editable

**Comment by jdamerow** [comment]
the word field should be filled. let's take the longest word of the selected wordnet concepts.

**Comment by mkarthik90** [comment]
@jdamerow Addressed review comments.

**Comment by jdamerow** [comment]
retest this please

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java [hunk 1]
```diff
@@ -115,14 +115,18 @@ public String prepareConceptWrapperAdd(HttpServletRequest req, ModelMap model) {
...
+        }
         if (req.getParameter("lists") == null || req.getParameter("lists").trim().isEmpty()) {
             req.setAttribute("errormsg", "You have to select a concept list.");
             return "forward:/auth/conceptlist/addconceptwrapper";
         }
         String[] wrappers = req.getParameter("wrapperids").split(Constants.CONCEPT_SEPARATOR);
 		if (wrappers.length > 0) {
 			ConceptEntry conceptEntry = new ConceptEntry();
-			conceptEntry.setWord(conceptManager.getConceptEntry(wrappers[0]).getWord().replace("_", " "));
+            conceptEntry.setWord(req.getParameter("word"));
```
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java]: Fixed it.

**Pull request merged**